src/bundle/Resources/views/content/tab/versions/table.html.twig

```diff
@@ -52,8 +52,8 @@
             {% if is_draft_conflict %}
                 <td class="ez-table__cell ez-table__cell--has-action-btns text-right">
                     {% set edit_draft_disabled = version.author.id != admin_ui_config.user.user.id %}
-                    <a href="{{ path('ez_content_draft_edit', { 'contentId': version.contentInfo.id, 'versionNo': version.versionNo, 'language': version.translations[0].languageCode, 'locationId': location.id }) }}"
-                       class="btn btn-icon mx-2 {% if edit_draft_disabled %}ez-btn--prevented{% endif %}"
+                    <a href="{{ path(edit_url, { 'contentId': version.contentInfo.id, 'versionNo': version.versionNo, 'language': version.translations[0].languageCode, 'locationId': location.id }) }}"
```

Comment:
It would be better to do generate needed URL and assign it to the variable to avoid issues when parameters to these routes change. They now have the same parameters but it might be changed sometime in the future.